packages/commercetools/api-client/src/index.ts

```diff
@@ -29,33 +31,46 @@ let country = '';
 let countries = [];
 let currencies = [];
 let locales = [];
+let currentToken: Token = null;
+let api: ApiConfig = null;
+let tokenEvents: TokenEvents = {
```

Comment:
imho it should be called `auth` so we can group all properties connected to auth here. `tokenEvents` tells absolutely nothing about what these properties are responsible for